Formula/zsxd.rb

```diff
@@ -1,9 +1,9 @@
 class Zsxd < Formula
   desc "Zelda Mystery of Solarus XD"
   homepage "https://www.solarus-games.org/en/games/the-legend-of-zelda-mystery-of-solarus-xd"
```

Comment:
```suggestion
  homepage "https://www.solarus-games.org/games/the-legend-of-zelda-mystery-of-solarus-xd/"
```

Reply:
You are right, somehow browser did the magic redirection.

Reply:
Yeah it was a 404 with a `meta http-equiv="refresh"` tag. Bit of an odd way to do it rather than a 301.